libsql-remote/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
```

Comment:
What is the value add of a new crate? If we are exposing things already in libsql as the main entry point, why would users benefit from this extra crate (extra link time spend). In addition, to achieve what you have here we need to duplicate a lot of code. Luckily, we should be able to reuse `Rows` and `Row` from libsql since it doesn't do any async work (meaning we avoid the trouble we have with Connection/Statement/Transaction).

Reply:
As far as I understand, having a separate crate makes it easier to maintain it. It is easier to ensure that a small separate create compiles for cloudflare than to ensure that the whole big umbrella crate compiles for cloudflare. Even with default-features = false.

Reply:
> As far as I understand, having a separate crate makes it easier to maintain it. 

This is not true, and we learned this in tokio hence why its one big crate right now. Dependencies get wild between them and there is really no added benefit to an additional crate.

>  It is easier to ensure that a small separate create compiles for cloudflare than to ensure that the whole big umbrella crate compiles for cloudflare. 

Somewhat true, but the work is done once and is easy to maintain if you have the right tools like a CI job that checks all the features. Also, if you write the feature based code in a certain why its not super invasive.